internal/schemautil/common.go

```diff
@@ -10,14 +10,14 @@ import (
 )
 
 //goland:noinspection GoDeprecation
-func GetACLUserValidateFunc() schema.SchemaValidateFunc {
+func GetACLUserValidateFunc() schema.SchemaValidateFunc { //nolint:staticcheck
```

Comment:
this says that we use deprecated schema.SchemaValidateFunc

I believe we shouldn't mute deprecation issues